src/main/java/uk/gov/hmcts/reform/blobrouter/services/EnvelopeService.java

```diff
@@ -63,7 +63,7 @@ public EnvelopeService(
     public UUID createNewEnvelope(String containerName, String blobName, Instant blobCreationDate) {
         UUID id = envelopeRepository
             .insert(
-                new NewEnvelope(containerName, blobName, blobCreationDate, null, Status.CREATED)
+                new NewEnvelope(containerName, blobName, blobCreationDate, null, Status.CREATED, null)
```

Comment:
are you going to add filesize In next pr ?

Reply:
yes, this PR is only repo